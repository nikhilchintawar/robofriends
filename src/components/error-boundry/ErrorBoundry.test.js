import { shallow } from 'enzyme';
import React from 'react';

import ErrorBoundry from './ErrorBoundry';

it('testig error boundry', () => {
    expect(shallow(<ErrorBoundry />)).toMatchSnapshot()
})