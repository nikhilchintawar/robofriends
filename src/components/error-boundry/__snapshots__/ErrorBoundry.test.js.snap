// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testig error boundry 1`] = `ShallowWrapper {}`;
